external-crates/move/crates/move-analyzer/trace-adapter/src/runtime.ts

```diff
@@ -217,81 +217,78 @@ export interface IMoveCallInfo {
 }
 
 /**
- * Describes the type of PTB summary.
+ * Describes the type of external event summary,
+ * a Move call info descrption or a string representing
+ * another external event.
  */
-export type PTBSummary = IMoveCallInfo | string;
+export type ExtEventSummary = IMoveCallInfo | string;
 
 /**
- * Describes PTB frame containing the PTB summary.
+ * Describes a frame containing a summary of all external events.
  */
-interface IPTBSummaryFrame {
+interface IExtEventsSummaryFrame {
     /**
-     * PTB summary frame ID.
+     * Summary frame ID.
      */
     id: number;
     /**
-     * PTB summary frame (virtual) line.
+     * Summary frame (virtual) line.
      */
     line: number;
     /**
-     * PTB summary.
+     * Name of the summary frame in camel case.
      */
-    summary: PTBSummary[];
+    camel_case_name: string;
```

Comment:
Small nit, but it feels weird to have both of these vs one and then converting between the two?